vars/perfRegressionParallelPipeline.groovy

```diff
@@ -230,6 +230,29 @@ def call(Map pipelineParams) {
                                                 }
                                             }
                                         }
+                                        stage("Send email for ${sub_test}") {
+                                            catchError(stageResult: 'FAILURE') {
+                                                script {
+                                                    wrap([$class: 'BuildUser']) {
+                                                        dir('scylla-cluster-tests') {
+                                                            def email_recipients = groovy.json.JsonOutput.toJson(params.email_recipients)
+
+                                                            sh """
+                                                            #!/bin/bash
+
+                                                            set -xe
+                                                            env
+
+                                                            test_id=`cat latest/test_id`
+                                                            echo "start send-email stage ..."
+                                                            ./docker/env/hydra.sh send-email --test-id "`cat latest/test_id`" --email-recipients $email_recipients --logdir "`pwd`"
```

Comment:
you have variable `test_id`, you can use it in command,

Reply:
where do i have `test_id` variable?
in the environment variables it doesn't exist, and in the groovy file there is no declaration of it...
if we do, please, point it out for me please

Reply:
then this line is not needed i think `test_id=`cat latest/test_id`

Reply:
removing